sdk_container/src/third_party/coreos-overlay/sys-apps/systemd/systemd-255.4.ebuild

```diff
@@ -105,6 +107,8 @@ PEFILE_DEPEND='dev-python/pefile[${PYTHON_USEDEP}]'
 # flag). The image stage fails with "Failed to resolve
 # typeattributeset statement at
 # /var/lib/selinux/mcs/tmp/modules/400/ntp/cil:120"
+#
+# Flatcar: Added a dep on sys-apps/kbd. TODO: Why?
```

Comment:
@ader1990: Could you give me a pointer about reasons for adding sys-apps/kbd to dependencies? I'll update this comment then.

Reply:
sys-apps/kbd contains the loadkeys binary, required by the dracut module https://github.com/dracutdevs/dracut/blob/master/modules.d/10i18n/module-setup.sh, which in turn is required by the dracut module of systemd - vconsole setup https://github.com/dracutdevs/dracut/blob/master/modules.d/00systemd/module-setup.sh#L45 and https://github.com/dracutdevs/dracut/blob/master/modules.d/00systemd/module-setup.sh#L106

Reply:
More info here: https://github.com/flatcar/scripts/pull/1679/commits/fcfbf18b8f38f99800d7c81bb68552147cae6f0b

Reply:
and here: https://github.com/flatcar/bootengine/pull/87